classes/Task/Update/UpdateComplete.php

```diff
@@ -79,6 +79,17 @@ public function run(): int
         }
 
         // removing temporary files
+        $tmpModulePath = $this->container->getProperty(UpgradeContainer::TMP_MODULES_PATH);
+        if ($this->container->getFileSystem()->exists($tmpModulePath)) {
+            foreach (scandir($tmpModulePath) as $item) {
+                if ($item !== '.' && $item !== '..' && $item !== 'index.php') {
```

Comment:
I've seen these checks on another place, I wonder is a helper method for this is a good addition to the project.